### PR TITLE
Private Key added to memory when Button held

### PR DESCRIPTION
**Pull request by MillerApps**
Made the private Key and QR Code generate upon holding the button.

### CODE REVIEW

File: Trust/Export/ViewControllers/ExportPrivateKeyViewController.swift [hunk 1]
```diff
@@ -93,21 +93,21 @@ class ExportPrivateKeyViewConroller: UIViewController {
...
-        createQRCode()
     }
 
     func blur(image: UIImageView) {
-        let blur = UIBlurEffect(style: .extraLight)
+        let blur = UIBlurEffect(style: .dark)
         let view = UIVisualEffectView(effect: blur)
 
-        view.frame = image.bounds
+        view.frame = CGRect(x: -130, y: -5, width: Layout.widthAndHeight, height: 10) // Needed to postion blur in corrct space, setting to image.bounds = 0,0,0,0, thus the postion is incorrect.
```
Comment (vikmeup) [inline Trust/Export/ViewControllers/ExportPrivateKeyViewController.swift]: Do not hardcode any pixel values.

Comment (MillerApps) [reply Trust/Export/ViewControllers/ExportPrivateKeyViewController.swift]: Fixed using auto layout

**Comment by vikmeup** [comment]
@MillerApps did you look into different wallets, how do they do private key reveal?

**Comment by MillerApps** [comment]
@vikmeup I didn’t look at any iOS wallets, I did look at MEW and Metamask though. MEW shows a blacked area with a clickable bit of text that says show which reveals a QR Code, they also have an option to copy the private key as text. Metamask has an option to copy the private key as text.

**Comment by MillerApps** [comment]
@vikmeup Your thoughts?

**Comment by vikmeup** [comment]
Should be ok for now, I don't really like show this view looks like tho. I will refine this later.

**Comment by MillerApps** [comment]
Sounds good, I'm not not a fan of the UI for it either but, I couldn't think of any alternatives.

**Pull request merged**